module.exports.handleError = (err) => {
  console.log(err)
  process.exit(1);
}